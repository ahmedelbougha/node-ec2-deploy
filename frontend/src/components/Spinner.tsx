import { StyledSpinner } from './styles/Spinner.styled';
import { Container } from './styles/Container.styled';

export default function Spinner() {
  return (
    <Container>
      <StyledSpinner />
    </Container>
  );
}
