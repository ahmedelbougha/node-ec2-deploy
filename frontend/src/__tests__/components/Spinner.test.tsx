import renderer from 'react-test-renderer';
import Spinner from '../../components/Spinner';

describe('Spinner', () => {
  it('renders', () => {
    const component = renderer.create(<Spinner />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
