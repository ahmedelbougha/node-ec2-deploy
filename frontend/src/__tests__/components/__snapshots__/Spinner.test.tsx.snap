// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders 1`] = `
<div
  className="sc-gswNZR lhJxqP"
>
  <div
    className="sc-bcXHqe exUSDF"
  />
</div>
`;
